pmpro-extra-expiration-warning-emails.php

```diff
@@ -204,7 +204,19 @@ function pmproeewe_extra_emails() {
 			
 			if ( !empty( $euser ) ) {
 				
-				$euser->membership_level = pmpro_getMembershipLevelForUser( $euser->ID );
+				if(pmprommpu_is_loaded()) {

```

Comment:
Do we need a function_exists check here?

Reply:
Or we don't even have to check. The pmpro_getMembershipLevelsForUser() function is in core PMPro. So just use that and do something special if there are > 1 level.